### ISSUE TITLE
Default validator allows trailing `@`

### ISSUE THREAD

**Issue by prognostikos** [substancelab/activemodel-email_address_validator#18]
Looks like String#split (used [here](https://github.com/substancelab/activemodel-email_address_validator/blob/master/lib/activemodel_email_address_validator/email_address.rb#L29)) ignores trailing, but not leading, characters.

```irb
[67] dev:(main)> "@user@host.domain@@@".split("@")
=> ["", "user", "host.domain"]
[68] dev:(main)> "user@host.domain@@@".split("@")
=> ["user", "host.domain"]
```

Ruby 2.7.5, Rails 6.1.4.1

**Comment by koppen** [comment]
Thanks, much appreciated! Honestly, I never knew that, but it sure is well documented. Fixed in #24 👍

**Comment by prognostikos** [comment]
@koppen thank you for such a quick fix 🎉
